WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -888,9 +888,11 @@ - (void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withT
                                  completion:nil];
 }
 
+// this method allows this VC to be a valid unwind destination for this selector
 - (IBAction)unwindOutWithSegue:(UIStoryboardSegue *)segue
 {
-    // here so that this VC is a valid unwind destination for this selector
+    // unwind segues don't seem to always clean themselves up 😫
+    [self dismissViewControllerAnimated:NO completion:nil];
```

Comment:
Do we need to double check which segue it is, or that something's being presented?